### DISCUSSION TITLE
will there be a mod in Russian?

### DISCUSSION THREAD

**Discussion by elwood2** [Monika-After-Story/MonikaModDev#8772] [general - translations]
will there be a mod in Russian?

**Comment by MrVavlo** [comment]
[#3761](https://github.com/Monika-After-Story/MonikaModDev/issues/3761)
[#3826](https://github.com/Monika-After-Story/MonikaModDev/issues/3826)

...Except they're outdated and those people don't work on translation anymore, from what I see.

Your best bet is RG Smoking Room team, as far as I know

**Reply by Akh0r** [reply]
I can do the translation.
I have already done it back in 2019, but the team I was working for ceased to exist, leaving me only with my RenPy code.
If anyone tells me what needs to be done to upload the translation and to actually implement it into the game (simply overwrite script files?) - then I can spend a couple of weeks and translate the rest of the text.

___

Я могу перевести.
Я уже делал это в 2019-м, но команда, на которую я тогда работал, развалилась, и я остался только с кучей кода на РенПае. 
Если бы мне кто-нибудь рассказал, как доделать перевод и запустить его в игре (просто заменить файлы скриптов?) – я бы потратил пару недель и перевёл бы остаток текста.

Конечно, если мы с тобой тут не одни, кому это интересно :)

**Reply by Inui-senpai** [reply]
RGSR already updated their translation up to 0.12.11. No one is interested in your offer anymore. Sorry no sorry.
